freqtrade/strategy/interface.py

```diff
@@ -69,6 +69,10 @@ class IStrategy(ABC, HyperStrategyMixin):
     # associated stoploss
     stoploss: float
 
+    # custom order price
+    entryprice: Optional[float] = None
+    exitprice: Optional[float] = None
```

Comment:
This will not work.
there is one strategy object for all of freqtrade - storing a price here will have that price apply to ALL pairs that are evaluated - and as such will lead to conditional errors (only when more than one order is open at the same time).

i also really don't see why you'd need this ...